maestro-ios-xctest-runner/build-maestro-ios-runner.sh

```diff
@@ -1,10 +1,14 @@
+#!/usr/bin/env sh
+
 ## Build the UI test
 ## TODO: make destination generic for iOS 15 simulator
 rm -rf ./build/Products || exit 1
 xcodebuild ARCHS="x86_64 arm64" \
   ONLY_ACTIVE_ARCH=NO \
   -project ./maestro-ios-xctest-runner/maestro-driver-ios.xcodeproj \
-  -scheme maestro-driver-ios -sdk iphonesimulator \
+  -scheme maestro-driver-ios \
+  -sdk iphonesimulator \
+  -destination "generic/platform=iOS Simulator" \
```

Comment:
I'm not sure if I understood what this does?

Reply:
Basically, be explicitly specifying the destination we avoid [this warning](https://github.com/mobile-dev-inc/maestro/pull/1830#issue-2431675024) (and are more deterministic).

This actually does what we want – we want a generic build for any simulator, not for a particular one

[more info](https://mokacoding.com/blog/xcodebuild-destination-options/)